env/env.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package env
+
+import (
+	"os"
+	"strconv"
+)
+
+// GetEnvAsStringOrFallback returns the env variable for the given key
+// and falls back to the given defaultValue if not set
+func GetEnvAsStringOrFallback(key, defaultValue string) string {
```

Comment:
I would either call it `GetEnvAsStringOrDefault` (rather than fallback), or simply `GetEnvAsStringOr`

Reply:
Why not just `GetEnvString(key, default string) string` and similar?

Reply:
Or even just `GetString` since it's already in an "env" package

Reply:
The origin PR is https://github.com/kubernetes/kubernetes/pull/20080,it's used in  pkg/volume/flocker and https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/rest/client.go#L140.I don’t know if it can be used elsewhere.